query/security/sts/src/main/java/org/codice/ddf/admin/security/sts/StsFieldProvider.java

```diff
@@ -33,18 +34,22 @@
 
     private ConfiguratorFactory configuratorFactory;
 
-    public StsActionCreator(ConfiguratorFactory configuratorFactory) {
+    private GetStsClaimsFunctionField getStsClaims;
+
+    public StsFieldProvider(ConfiguratorFactory configuratorFactory) {
         super(NAME, TYPE_NAME, DESCRIPTION);
         this.configuratorFactory = configuratorFactory;
+        getStsClaims = new GetStsClaimsFunctionField(configuratorFactory);
+        updateInnerFieldPaths();
     }
 
     @Override
-    public List<Action> getDiscoveryActions() {
-        return Arrays.asList(new GetStsClaimsAction(configuratorFactory));
+    public List<Field> getDiscoveryFields() {
+        return Arrays.asList(getStsClaims);
```

Comment:
Do we need to worry about the mutability of the discovery and mutation fields in the field provider classes?